packages/remix-react/browser.tsx

```diff
@@ -303,11 +299,14 @@ export function RemixBrowser(_props: RemixBrowserProps): ReactElement {
   // removed from a component, but the styles will still be present in the
   // server HTML. This allows our HMR logic to clear the critical CSS state.
   // eslint-disable-next-line react-hooks/rules-of-hooks
-  let [criticalCss, clearCriticalCss] = React.useReducer(
-    criticalCssReducer,
-    window.__remixContext.criticalCss
+  let [criticalCss, setCriticalCss] = React.useState(
```

Comment:
Coming back to this code, I found the use of `useReducer` confusing so opted for a simpler `useState` call.